src/Moryx.Orders/Operation.cs

```diff
@@ -108,13 +108,23 @@ public class Operation
         /// <summary>
         /// Current state classification of this operation
         /// </summary>
-        public virtual OperationClassification State { get => (OperationClassification)((int)FullState & 0xFF); protected set => FullState = value; }
+        public virtual OperationClassification State
+        {
+            get => (OperationClassification)((int)FullState & 0xFF); 
+            protected set => FullState = value;
+        }
 
         /// <summary>
         /// Current state classification of this operation
         /// </summary>
         public virtual OperationClassification FullState { get; protected set; }
 
+        /// <summary>
+        /// Detailed display name of the state
+        /// TODO: Remove this property in next major and replace rework OperationClassification
```

Comment:
Can you replace the Todo with an `[Obsolete(...)]` attribute? This way it's directly visible for all useres of the class 🤔